scpdt/impl.py

```diff
@@ -375,7 +376,7 @@ class DebugDTRunner(DTRunner):
     Almost verbatim copy of `doctest.DebugRunner`.
     """
     def run(self, test, compileflags=None, out=None, clear_globs=True):
-        r = super().run(test, compileflags, out, False)
+        r = super().run(test, compileflags=compileflags, out=out, clear_globs=False)
```

Comment:
kwargs do make it read much better, thank you!

Now that I look at this, why does it drop `clear_globs` argument? This is pre-existing of course, can we pass it here nonetheless? At this point, probably best to just add an `TODO` comment and move on though.